src/reaction.rs

```diff
@@ -558,7 +558,12 @@ Here's my footer -- bob@example.net"
     ) -> Result<()> {
         let event = t
             .evtracker
-            .get_matching(|evt| matches!(evt, EventType::ReactionsChanged { .. }))
+            .get_matching(|evt| {
+                matches!(
+                    evt,
+                    EventType::ReactionsChanged { .. } | EventType::IncomingMsg { .. }
```

Comment:
The `unreachable!()` below should be replaced by sth like `panic!("Unexepected event {event:?}")`